java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -3270,7 +3273,11 @@ private void setNativeNullCount(int nullCount) throws CudfException {
     }
 
     public DType getNativeType() {
-      return DType.fromNative(getNativeTypeId(getViewHandle()));
+      return DType.fromNative(getNativeTypeId(getViewHandle()), getNativeTypeScale(getViewHandle()));
+    }
+
+    public int getNativeScale() {
```

Comment:
Do we need this method, or at least does it need to be public?  Maybe I'm missing the use case where someone wants to access this without also calling `getNativeType` which provides the scale via the `DType` result.

Reply:
I had added this accessor method earlier but now i don't see any use case for it. i.e no reason to just get the scale. Will remove it.

Reply:
It should not matter because OffHeapState, which this is a part of, is protected.  All the other methods are public too, but I am also fine with changing it.